src/components/mermaidInputs/InputSelectWithLabelAndValidation/InputSelectWithLabelAndValidation.js

```diff
@@ -1,10 +1,11 @@
-import React from 'react'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
-import { InputRow, Select, HelperText, RequiredIndicator } from '../../generic/form'
+import { InputRow, Select, HelperText, LabelContainer, RequiredIndicator } from '../../generic/form'
 import { inputOptionsPropTypes } from '../../../library/miscPropTypes'
-
 import InputValidationInfo from '../InputValidationInfo/InputValidationInfo'
 import mermaidInputsPropTypes from '../mermaidInputsPropTypes'
+import { IconButton } from '../../generic/buttons'
+import { IconInfo } from '../../icons'
 
 const InputSelectWithLabelAndValidation = ({
```

Comment:
kinda jokin, kinda not, should we rename this component `InputSelectWithLabelValidationHelperText`?

Reply:
I feel like its a bit unnecessarily verbose (same if we update `InputWithLabelAndValidation`)